tools/layer-zero-example/contracts/WHBAR.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.22;
+
+contract WHBAR {
```

Comment:
nit: let's add a comment that this is based on WETH9 with updated solidity version and functions

Reply:
Done.